views/partials/banners/download_firefox_bar.hbs

```diff
@@ -1,7 +1,7 @@
 <div id="download-firefox-bar" class="download-firefox download-firefox-bar banner">
   <div class="section-wrapper">
     <div class="whole">
-      <p>Firefox Monitor is brought to you by the all-new Firefox. <a class="internal-link" href="?utm_source={{ UTM_SOURCE }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=download_bar">Download&nbsp;Firefox&nbsp;now&nbsp;&#187;</a>
+      <p>Firefox Monitor is brought to you by the all-new Firefox. <a class="internal-link" href="https://www.mozilla.org/firefox/new/?utm_source={{ UTM_SOURCE }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=download_banner" target="_blank" rel="noopener">Download Firefox now &#187;</a>
```

Comment:
FWIW, this changed from `&utm_content=download_bar` to `&utm_content=download_banner` (if that matters).